apps/frontend/src/app/3_organisms/GettingStartedPopup/GettingStartedPopup.tsx

```diff
@@ -103,6 +93,36 @@ export const GettingStartedPopup: FC<GettingStartedPopupProps> = ({
 
   return (
     <Dialog isOpen={isOpen}>
+      <DialogHeader
+        title={t(translations.zeroPage.gettingStartedPopup.dialogTitle)}
+      />
+      <div className={styles.dialog}>
+        <Heading className={styles.title} type={HeadingType.h1}>
+          {t(translations.zeroPage.gettingStartedPopup.title)}
+        </Heading>
+        <div className={classNames(styles.content, 'mb-0')}>
+          <div className={styles.itemWrapper}>
+            <div className={styles.image}>
+              <Icon size={50} icon={IconNames.MAIL} />
+            </div>
+            <div className={styles.texts}>
+              <Heading className={styles.name} type={HeadingType.h2}>
+                {t(translationBasePath.notification.title)}
+              </Heading>
+              <Heading className={styles.itemDescription} type={HeadingType.h3}>
+                {t(translationBasePath.notification.description)}
+              </Heading>
+              <Link
+                text={t(
+                  translations.zeroPage.gettingStartedPopup.buttons.learnMore,
+                )}
+                href={sovrynWikiLinks.notification}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
I'd say we should hide mobile pagination as it looks weird right now because there are only 2 items and there is only 1 page. What do you think @soulBit ?

<img width="502" alt="image" src="https://user-images.githubusercontent.com/14878713/224967632-922f5d0b-c137-43f4-bc00-7a14b824776b.png">

Reply:
agreed

Reply:
@pietro-maximoff pls hide it then